src/lib/parser.c

```diff
@@ -1764,7 +1764,7 @@ RdbStatus elementSet(RdbParser *p) {
                 else
                     CALL_HANDLERS_CB(p, NOP, RDB_LEVEL_DATA, rdbData.handleSetMember, binfoItem->ref);
 
-                updateElementState(p, ST_SET_NEXT_ITEM, 0);
+                updateElementStateIterative(p);
```

Comment:
how come i don't see such a change for the Hash type?

Reply:
right. thanks.